models/redis/redis.go

```diff
@@ -0,0 +1,67 @@
+package redis
+
+import (
+	"log"
+
+	"biilabs.com/mam-data-explorer/models/config"
+	"github.com/go-redis/redis"
+)
+
+func ConnectRedisDB() *redis.Client {
+	rdb := redis.NewClient(&redis.Options{
+		Addr:     config.ConfigHandler.GetString("REDIS_URL"),
+		Password: config.ConfigHandler.GetString("REDIS_PASSWORD"),
```

Comment:
Would you like to store the information in certain file?

Reply:
I think it is better to put all configuration together inside config.yml.